librosa/display.py

```diff
@@ -767,6 +768,13 @@ def specshow(
     thaat : str, optional
         If using `chroma_h` display mode, specify the parent thaat.
 
+    auto_scale : bool
+        Certain plots are automatically scaled for squared axes if `True` (default).
+
+        These plots include covariance, self-similarity, self-distance etc.
```

Comment:
This part should be a bit more precise.  Specshow doesn't technically know about covariance, self-similarity, etc; what it does know is axis scales and data dimensions.  We may as well make that clear to the user here, something like  

`axes will have 'equal' aspect if the horizontal and vertical dimensions cover the same extent and their types match.`

This also makes me think that `auto_scale` should be `auto_aspect`.

Reply:
Yes, I guess being explicit is the safest way. I thought about linking the example from `plot_display.py` here here but that might be headache to maintain. I will use your definition as is in the docstring.